worker_node/job/send.js

```diff
@@ -149,12 +149,13 @@ class Send extends Base {
     }
   }
 
-  makeBaseTransaction() {
+  makeBaseTransaction(gasPrice) {
     return {
       operation: {
         ...this.config.transactionOperation,
       },
       nonce: -1,
+      gas_price: gasPrice,
```

Comment:
minor) how about just using this.config.gasPrice || 0 here without passing it as an arg?

Reply:
Looks better! Done